common/compositor/compositorthread.cpp

```diff
@@ -104,6 +104,11 @@ bool CompositorThread::Draw(std::vector<DrawState> &states,
   draw_succeeded_ = true;
   tasks_lock_.unlock();
   Resume();
+
+  if (states_.empty() && media_states_.empty()) {
```

Comment:
if both the states are empty, will we ever do any drawing?
If not we could use this as a fast path and return true?

Reply:
in this case we shouldn't be doing any drawing

Reply:
So we can check this case as soon as we enter the method and return true?

Reply:
so far, their is no data to show this is needed. Holing on to this PR.